README.md

```diff
@@ -1,2 +1,88 @@
-# hexabot-plugin-chatgpt
-The OpenAI ChatGPT Plugin for Hexabot Chatbot / Agent Builder to enable the LLM RAG Capability
+# ChatGPT RAG Plugin for Hexabot Chatbot Builder
```

Comment:
```suggestion
# ChatGPT Plugin for Hexabot Chatbot Builder
```

Can we also rename the file to chatgpt.plugin.ts ?

Reply:
Sure we can :heavy_check_mark: